utils/s2n_init.c

```diff
@@ -80,11 +82,12 @@ static bool s2n_cleanup_atexit_impl(void)
     /* the configs need to be wiped before resetting the memory callbacks */
     s2n_wipe_static_configs();
 
-    bool a = s2n_result_is_ok(s2n_rand_cleanup_thread());
-    bool b = s2n_result_is_ok(s2n_rand_cleanup());
-    bool c = s2n_mem_cleanup() == 0;
+    bool a = s2n_result_is_ok(s2n_locking_cleanup());
+    bool b = s2n_result_is_ok(s2n_rand_cleanup_thread());
+    bool c = s2n_result_is_ok(s2n_rand_cleanup());
+    bool d = s2n_mem_cleanup() == 0;
 
-    return a && b && c;
+    return a && b && c && d;
```

Comment:
Good that the alphabet is long :)

Reply:
You have shamed me into rewriting this instead of continuing down the alphabet.